upfadapter/pfcp/udp/message.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2024 Canonical Ltd.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// The content of this file was taken from https://github.com/omec-project/pfcp
```

Comment:
Do we need/want to keep this line?

Reply:
I addressed this in a different comment, let's discuss it there.